custom_model_runner/datarobot_drum/drum/adapters/drum_cli_adapter.py

```diff
@@ -27,15 +27,15 @@
 
 class DrumCLIAdapter(object):
     """
-    This class acts as a layer between the DRUM CLI and running a custom task's fit method. It will convert the
-    cli arguments (mostly paths) into tangible variables to be passed into fit.
+    This class acts as a layer between DRUM CLI/MLPiper and running a custom task's fit/predict method. It will convert
```

Comment:
I'll probably end up refactoring this class to have separate adapters for Fit/Score/Server that inherit from the same base class